freshpaper.py

```diff
@@ -215,10 +215,85 @@ def download_image_nasa(download_dir, image_extension="jpg"):
         log.error("Something went wrong..\nMaybe Internet is not working...")
         raise ConnectionError
 
+def download_image_unsplash_daily(download_dir, image_extension="jpg"):
+    """
+    Download & save the image
+    :param download_dir: directory where to download the image
+    :param image_extension: directory where to download the image
+    :return: downloaded image path
+    """
+    # mkt(s) HIN, EN-IN
+
+    url = "https://source.unsplash.com/daily"
+
+    try:
+        image_url = url
+
+        image_name = 'unsplash_daily'
+
+        date_time = datetime.now().strftime("%d_%m_%Y")
+        image_file_name = "{image_name}_{date_stamp}.{extention}".format(
+            image_name=image_name, date_stamp=date_time, extention=image_extension
+        )
+
+        image_path = os.path.join(os.sep, download_dir, image_file_name)
+        log.debug("download_dir: {}".format(download_dir))
+        log.debug("image_file_name: {}".format(image_file_name))
+        log.debug("image_path: {}".format(image_path))
+
+        if os.path.isfile(image_path):
+            log.info("No new wallpaper yet..updating to latest one.\n")
+            return image_path
+
+        log.info("Downloading...")
+        urlretrieve(image_url, filename=image_path)
+        return image_path
+    except URLError:
+        log.error("Something went wrong..\nMaybe Internet is not working...")
+        raise ConnectionError
+
+def download_image_unsplash_random(download_dir, image_extension="jpg"):
+    """
+    Download & save the image
+    :param download_dir: directory where to download the image
+    :param image_extension: directory where to download the image
+    :return: downloaded image path
+    """
+    # mkt(s) HIN, EN-IN
```

Comment:
we don't need this comment, it was for bing only

Reply:
OK, removed